pages/1.10/security/ent/tls-ssl/ca-custom/index.md

```diff
@@ -8,26 +8,321 @@ excerpt:
+
+The public key associated with the custom CA certificate has to be of type RSA.
+
+## ca\_certificate\_key\_path
+Path to a file within the `genconf/` directory containing the private key corresponding to the custom CA certificate, encoded in the OpenSSL (PKCS#8) PEM format. For example: `genconf/CA_cert.key`.
+
+**Note**: this is highly sensitive data. The configuration processor accesses this file only for configuration validation purposes, and does not copy the data. After successful configuration validation this file needs to be placed out-of-band into the file system of all DC/OS master nodes to the path /var/lib/dcos/pki/tls/CA/private/custom_ca.key before most DC/OS systemd units start up. The file has to be readable by the root user, and should have have 0600 permissions set.
+
+Required if `ca_certificate_path` is specified.
+
+## ca\_certificate\_chain\_path
+Path to a file within the `genconf/` directory containing the complete CA certification chain required for end-entity certificate verification, in the OpenSSL PEM format. For example: genconf/CA_cert_chain.pem.
+Has to be left undefined if `ca_certificate_path` points to a root CA certificate.
+Required if ca_certificate_path is specified and if the custom CA certificate is an *intermediate CA certificate*. 
+
+This needs to contain all CA certificates comprising the complete sequence starting precisely with the CA certificate that was used to sign the custom CA certificate and ending with a root CA certificate (where issuer and subject are equivalent), yielding a gapless certification path. The order is significant and the list has to contain at least one certificate.
+
+
+# Installing DC/OS Enterprise with a custom CA certificate
+## Starting point
+Based on the requirements described above, this is the starting point for the installation:
+
+- The installation of DC/OS Enterprise via the Advanced Installer has been prepared according to the corresponding [documentation](/1.10/installing/ent/custom/advanced/).
+
+- On the bootstrap node, the files carrying custom CA certificate, the associated private key and, optionally, the CA certificate chain have been placed into the `$DCOS_INSTALL_DIR/genconf/` directory (see the [section](#glossary-of-the-terms-used-in-this-documentation) above for more detailed description), Example (commands executed on the bootstrap node):
+
+```bash
+cd $DCOS_INSTALL_DIR
+ls genconf/
+```
+```bash 
+dcos-ca-certificate.crt
+dcos-ca-certificate-key.key
+dcos-ca-certificate-chain.crt
+```
+
+- The custom CA private key has been securely placed on all DC/OS master nodes and is sufficiently protected (refer to [this section](#manually-placing-the-private-key-associated-with-the-custom-ca-certificate-onto-the-master-nodes) for more details). Example (command issued on one of the DC/OS master nodes):
+
+```bash
+stat /var/lib/dcos/pki/tls/CA/private/dcos-ca-certificate-key.key
```

Comment:
Putting a private key onto this location would work. A private key has to be always on ``/var/lib/dcos/pki/tls/CA/private/custom_ca.key` path. I think we should update this section to not confuse users.

Reply:
... would *not* work. :)